src/components/IconButton/IconButton.tsx

```diff
@@ -122,77 +184,14 @@ const IconButton = forwardRef(
   }
 );
 
-IconButton.sizes = SIZES;
-IconButton.kinds = Button.kinds;
-IconButton.colors = Button.colors;
-
-IconButton.propTypes = {
-  /**
-   * id to be added to the element
-   */
-  id: PropTypes.string,
-  /**
-   * callback function when clicking the icon button
-   */
-  onClick: PropTypes.func,
-  /**
-   * class to be added to the button
-   */
-  className: PropTypes.string,
-  /**
-   * class to be added to the button wrapper
-   */
-  wrapperClassName: PropTypes.string,
-  /**
-   * Icon to be rendered
-   */
-  icon: PropTypes.oneOfType([PropTypes.string, PropTypes.node, PropTypes.element, PropTypes.func]),
-  /**
-   * a11y property to be added, used for screen reader to know what kind of button it is
-   */
-  ariaLabel: PropTypes.string,
-  /**
-   * Size of the icon
-   */
-  size: PropTypes.oneOf([
-    IconButton.sizes.XXS,
-    IconButton.sizes.XS,
-    IconButton.sizes.SMALL,
-    IconButton.sizes.MEDIUM,
-    IconButton.sizes.LARGE
-  ]),
-  /**
-   * Whether the tooltip should be displayed or not
-   */
-  hideTooltip: PropTypes.bool,
-  /**
-   * Tooltip wraps the button icon, it will display in the tooltip, if not present the aria label will be shown
-   */
-  tooltipContent: PropTypes.string,
-  /**
-   * Kind of button - like <Button />
-   */
-  kind: PropTypes.oneOf([IconButton.kinds.PRIMARY, IconButton.kinds.SECONDARY, IconButton.kinds.TERTIARY]),
-  /** The button's color  */
-  color: PropTypes.oneOf([
-    Button.colors.PRIMARY,
-    Button.colors.NEGATIVE,
-    Button.colors.POSITIVE,
-    Button.colors.ON_PRIMARY_COLOR,
-    Button.colors.ON_INVERTED_BACKGROUND
-  ]),
-  /**
-   * disabled state
-   */
-  disabled: PropTypes.bool,
-  /**
-   * if disabled - this will be shown in the tooltip
-   */
-  disabledReason: PropTypes.string,
-  dataTestId: PropTypes.string,
-  /** Change the focus indicator from around the button to within it */
-  insetFocus: PropTypes.bool
-};
+// Button.sizes = SIZES;
+// IconButton.kinds = Button.kinds;
```

Comment:
Awesome work, @SumeetHaryani
Can you please remove the commented out part of the code

Reply:
Thanks @orrgottlieb 
Removed commented code